admin/client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -134,27 +134,31 @@ class CampaignAdminList extends SilverStripeComponent {
       );
     });
 
-    // Get preview details
-    const classNames = itemLinks
-      ? 'cms-content__split cms-content__split--left-sm'
-      : 'cms-content__split cms-content__split--none';
+    // Set body
+    const body = accordionBlocks.length
+      ? (<Accordion>{accordionBlocks}</Accordion>)
+      : (
+        <div className="alert alert-warning" role="alert">
+          <strong>This campaign is empty.</strong> You can add pages by selecting{' '}
+          <em>Add to campaign</em> from within the <em>More Options</em> popup on{' '}
+          the edit page screen.
+        </div>
+      );
 
     return (
-      <div className={classNames}>
+      <div className="cms-content__split cms-content__split--left-sm">
```

Comment:
Before we were hiding the preview, but now if there's no preview, we leave the panel open, but display a message if there is no preview available.